src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -152,6 +157,7 @@
 import net.minecraftforge.registries.GameData;
 import net.minecraftforge.registries.IRegistryDelegate;
 
+import org.antlr.v4.runtime.misc.MultiMap;
```

Comment:
Is this being used?

Reply:
Oh, you can blame me for that one. Was importing Multimap (the one Mojang uses) and clicked the wrong import. In my normal workspaces I auto clean up imports on save, but I have that disabled when I work on Forge to prevent unneeded patches